### ISSUE TITLE
Remove process.exit handles (at least for usage as a module)

### ISSUE THREAD

**Issue by scic** [highcharts/node-export-server#20]
In lib/phantompool.js there are calls to **process.exit()** that terminate the server. When integrating this package as a node module in a server we wan't to control the termination of the server ourselves.

The calls mentioned:
```
process.on('SIGINT', function (code) {
    process.exit(code);
});

process.on('uncaughtException', function (e) {
    console.log('uncaughtException:', e);
    process.exit(0);
});
```

I think the SIGINT handle could be replaced with a doTerminate call like the exit handle.
The uncaughtException handle, could be removed completely.

**Comment by tonylukasavage** [comment]
I agree wholeheartedly. This was cryptically circumventing our unit testing. Get rid of it.

**Comment by cvasseng** [comment]
Firstly, sorry for the delay in addressing this.

I agree that it's bad practice.

Unfortunately it's the only way I've found that consistently terminates the Phantom-processes, both on error conditions and on normal exists.

I've added a compromise until a better solution can be found: setting `listenToProcessExits` to `false` in the the `initPool` config object will skip attaching the event listeners to the process.

Beware though that this _may_ lead to zombie processes both if the service crashes, and when it's exited normally.
